pandas/tests/io/test_gbq.py

```diff
@@ -144,6 +144,24 @@ def mock_read_gbq(sql, **kwargs):
     assert "use_bqstorage_api" not in captured_kwargs
 
 
+@pytest.mark.parametrize("progress_bar", [None, "foo"])
+def test_read_gbq_progress_bar_type_kwarg(monkeypatch, progress_bar):
+    # GH 29857
+    captured_kwargs = {}
+
+    def mock_read_gbq(sql, **kwargs):
+        captured_kwargs.update(kwargs)
+        return DataFrame([[1.0]])
+
+    monkeypatch.setattr("pandas_gbq.read_gbq", mock_read_gbq)
+    pd.read_gbq("SELECT 1", progress_bar_type=progress_bar)
+
+    if progress_bar:
```

Comment:
Can this test be simplified to just call with the argument? Some of the monkey patching seems to be testing non-pandas behavior

Reply:
this is ok, its the same type of test we use for other kwargs